action.yml

```diff
@@ -85,6 +85,16 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - name: Install Binaries
+      shell: bash
+      run: |
+        if [[ "${{runner.os}}"  == "Linux" ]]; then
+          sudo apt-get install clang-format-${{ inputs.version }} clang-tidy-${{ inputs.version }}
```

Comment:
The CI is failing to find v13 & v14 in the apt pkgs list. Maybe, it would help to run `apt-get update` beforehand?

Reply:
Yeah, we could use a shell script located in the action repo. I didn't know about the composite steps not supporting `if` conditions.